README.md

```diff
@@ -63,4 +68,63 @@ npm install -g webpack-cli
   - npm run seed
 7) Check if the tables populated
   - psql reviews
-  - reviews=# select * from reviews;
\ No newline at end of file
+  - reviews=# select * from reviews;
+
+## API
+
+### Restaurant Impression
+
+#### HTTP request
+
+GET http://127.0.0.1:3010/:id/impression
```

Comment:
I get a 500 status error to this route, but the route for /reviews is successful and I was able to see the data come back.

Reply:
Did you use a number between 1 and 5 as the restaurant id?